common/xrdp_client_info.h

```diff
@@ -143,7 +143,7 @@ struct xrdp_client_info
   int use_frame_acks;
   int max_unacknowledged_frame_count;
 
-  int disableSSLv3; /* 0 = no, 1 = yes */
+  long ssl_protocols;
```

Comment:
We would need to check which members are actually used by X11rdp and xorgxrdp. X11rdp is important because it can be compiled separately, perhaps from older sources. I feel this discussion is more suitable for a separate issue.